projecteuler/helpers/primenumbers.py

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env python3
+
+def get_prime_numbers(min_number, max_number):
```

Comment:
I'm going to assume you know that this method is terrible :D

Reply:
I've rewritten it now. Funny how often the word number was in that method :D